src/main/java/com/github/dbmdz/solrocr/solr/OcrHighlightParams.java

```diff
@@ -15,4 +20,40 @@ public interface OcrHighlightParams extends HighlightParams {
   String TIME_ALLOWED = "hl.ocr.timeAllowed";
   String ALIGN_SPANS = "hl.ocr.alignSpans";
   String TRACK_PAGES = "hl.ocr.trackPages";
+
+  /**
+   * Get a boolean value from a `hl.ocr.*` parameter. If no value is given for the parameter, try to
+   * get the value from the corresponding `hl.*` parameter, otherwise return the default.
+   */
+  static boolean getBool(SolrParams params, String name, boolean defaultValue) {
+    Boolean value = params.getBool(name);
+    if (value == null) {
+      return params.getBool(name.replace("hl.ocr", "hl"), defaultValue);
+    }
+    return value;
+  }
+
+  /**
+   * Get a String value from a `hl.ocr.*` parameter. If no value is given for the parameter, try to
+   * get the value from the corresponding `hl.*` parameter.
+   */
+  static String get(SolrParams params, String name) {
+    String value = params.get(name);
+    if (value == null) {
+      return params.get(name.replace("hl.ocr", "hl"));
+    }
+    return value;
+  }
+
+  /**
+   * Get a String value from a `hl.ocr.*` parameter. If no value is given for the parameter, try to
+   * get the value from the corresponding `hl.*` parameter, otherwise return the default.
+   */
+  static String get(SolrParams params, String name, String defaultValue) {
+    String value = OcrHighlightParams.get(params, name);
+    if (value == null) {
+      return defaultValue;
```

Comment:
The docstring says that if no value is given for `hl.ocr.*`, the value from `hl.*` will be tried before returning the default. I don't see this here, is this correct?

Reply:
It's a bit hidden in this `get` overload, but the lookup for the `hl.*` equivalent happens in the other `get` variant that gets called in the first line of the method.
This overload is simply a variant that returns a default value if neither the `hl.ocr.*` or the `hl.*` variant has a value set.

Reply:
Oh right, my mistake. Thanks!